scripts/check_nightly.py

```diff
@@ -62,7 +62,8 @@ def nightly_tests():
     ret = set()
     for test in tests:
         t = test.strip().split(' ')
-        if t[0] == 'expensive':
+        if t[0] == 'expensive' or (t[0] == '#' and t[1] == 'expensive'):
+            # It's okay to comment out a very expensive test intentionally
```

Comment:
But `test_all_chunks_accepted_1000_slow` is not commented out above, it's removed, how does it work for it?

Reply:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/13259400/83454228-bf0db500-a410-11ea-876b-1fe02246a253.png">
There's a commented one above

Reply:
Finally finished, tooked 2h52min locally 😄